src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figures {
+    private final double radius;
+
+    public Circle(double radius, String color) {
+        this.setColor(color);

```

Comment:
you should use the constructor of the Figure for this purpose, the same for other figures